sqlup-mode.el

```diff
@@ -144,9 +144,8 @@ Other than <RET>, characters are in variable sqlup-trigger-characters."
 (defun sqlup-match-eval-keyword-p (dialect)
   "Return t if the code just before point ends with an eval keyword valid in
 the given DIALECT of SQL."
-  (cl-some 'identity
-           (mapcar #'(lambda (kw) (looking-back (concat kw "[\s\n\r\t]*") 0))
-                   (cdr (assoc dialect sqlup-eval-keywords)))))
+  (cl-some (lambda (kw) (looking-back (concat kw "\\s-*") 0))
```

Comment:
The difference between `cl` and `seq` is that `cl` is part of GNU emacs. I'd rather avoid external dependencies at this time.